examples/00_bmg_gemm/00_bmg_gemm.cpp

```diff
@@ -368,8 +370,11 @@ int main(int argc, const char** argv)
 
   // For Intel BMG, PipelineStages defines how many k-blocks ahead to prefetch from A and B.
   constexpr int PipelineStages = 2;
-  using GEMMDispatchPolicy = cutlass::gemm::MainloopIntelXeXMX16<PipelineStages>;
-  using EpilogueDispatchPolicy = cutlass::epilogue::IntelXeXMX16;
+   // For older version of copy/mma atom, use cutlass::gemm::MainloopIntelXeXMX16 as dispatch policy for CollectiveMma 
+   // and use cutlass::epilogue::IntelXeXMX16 as dispatch policy for CollectiveEpilogue
+  // use cutlass::epilogue::IntelXeXMX16 as dispatch policy for CollectiveEpilogue
```

Comment:
Since we want to deprecate/remove the old atoms, let's not worry about explaining to people how to use them here.